model/testing/src/icon4py/model/testing/pytest_config.py

```diff
@@ -7,25 +7,28 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 import os
+from collections.abc import Callable
+from typing import Final
 
 import pytest
 from gt4py.next import gtfn_cpu, gtfn_gpu, itir_python
 
-from icon4py.model.common.test_utils.datatest_utils import (
+from icon4py.model.testing.datatest_utils import (
     GLOBAL_EXPERIMENT,
     REGIONAL_EXPERIMENT,
 )
 
 
-DEFAULT_BACKEND = "roundtrip"
+DEFAULT_BACKEND: Final = "roundtrip"
 
-backends = {
+BACKENDS: dict[str, Callable] = {
```

Comment:
strictly they are not constant because the get updated later with the DaCe backends.